x-pack/solutions/security/plugins/security_solution/public/assistant/get_comments/index.tsx

```diff
@@ -148,7 +146,7 @@ export const getComments: GetAssistantMessages = ({
         timelineAvatar: isUser ? (
           <UserAvatar />
         ) : (
-          <EuiAvatar name="machine" size="l" color="subdued" iconType={AssistantAvatar} />
+          <AssistantAvatar name="machine" size="l" color="subdued" />
```

Comment:
There appears to be a subtle difference between `EuiAvatar` and `AssistantAvatar`, such that:

`EuiAvatar` did not appear to have a border around the avatar, as illustrated by the following screenshot:

![conversation_day_before](https://github.com/user-attachments/assets/f4a9a139-7d2e-48ec-828e-e0036ff4db18)

`AssistantAvatar` appears to have a border around the `AssistantAvatar`, as illustrated by the following screenshot:

![conversation_day_pr](https://github.com/user-attachments/assets/08609e56-8302-4afc-beec-096d7b4f240a)

Reply:
That's correct.  The original `AssistantAvatar` component that was replaced with _this_ avatar had a border, so I added it.

To your point, in this context, it makes no sense.  I'll make that border optional.